src/VocabGameData.js

```diff
@@ -18,6 +18,16 @@ export function createDataset(base) {
   if (!Array.isArray(base)) {
     throw new Error('createDataset does not allow non arrays')
   }
+  base.forEach(function(array) {
+    if (array.includes(null)) {
+      throw new Error ("null values");
+    }
+  });
+  let baseFlattened = [].concat(...base);
+  let set = new Set(baseFlattened);
+  if (set.size !== baseFlattened.length) {
+    throw new Error ("duplicate value");
```

Comment:
so the test unit will look for "duplicate value" within the sentence that you write. it's a partial word match. 

so, would you add more detail to the error message because this is how it will render 
![image](https://user-images.githubusercontent.com/29721784/50577398-5154f800-0ddc-11e9-8c67-40f5b5b9e8da.png)

so more context in the error message would be helpful!
this applies to the other error messages you wrote.

Reply:
Oh I see, I thought it needed to match the error you gave me exactly lol. I'll change that and push again in a bit, thanks!